setup.py

```diff
@@ -39,38 +39,38 @@
     package_dir={"": "polaris"},
     packages=find_packages("polaris"),
     install_requires=[
-        "aiohttp==3.6.3",
+        "aiohttp==3.7.3; python_version >= '3.6'",
         "aiohttp-sse-client==0.1.7",
-        "async-timeout==3.0.1",
-        "attrs==20.2.0",
-        "certifi==2020.6.20",
+        "async-timeout==3.0.1; python_full_version >= '3.5.3'",
+        "attrs==20.3.0; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
+        "certifi==2020.11.8",
         "cffi==1.14.3",
         "chardet==3.0.4",
         "crc16==0.1.1",
-        "cryptography==3.1.1",
-        "django==2.2.16",
+        "cryptography==3.2.1",
+        "django==2.2.17",
         "django-cors-headers==3.5.0",
         "django-environ==0.4.5",
         "django-model-utils==4.0.0",
-        "djangorestframework==3.12.1",
-        "idna==2.10",
+        "djangorestframework==3.12.2",
+        "idna==2.10; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
         "mnemonic==0.19",
-        "multidict==4.7.6",
+        "multidict==5.0.2; python_version >= '3.5'",
```

Comment:
i was concerned about the major version update but it looks like we'll be fine:
https://multidict.readthedocs.io/en/stable/changes.html